test/packages/good/elasticsearch/transform/metadata_united/transform.json

```diff
@@ -0,0 +1,41 @@
+{
+    "source": {
+        "index": [
+            "metrics-endpoint.metadata_current_default*",
+            ".fleet-agents*"
+        ]
+    },
+    "dest": {
+        "index": ".metrics-endpoint.metadata_united_default"
```

Comment:
Will referenced indexes be managed by Fleet when installing the package?

Reply:
We could mention this in the description of the file in the spec.

Reply:
I have also created https://github.com/elastic/package-spec/issues/354, it may be good to have some reference docs about how the different resources in a package should be interpreted.

Reply:
@jsoriano I think your link for "description of the file" may be wrong. Could you clarify where this should be documented for now?

Reply:
Oops, multitasking is bad :sweat_smile: This is the correct link I wanted to use: https://github.com/elastic/package-spec/pull/307/files#diff-a60684d31c8050dde697084c6f224112dad180edd8df8fe820c8deea2747b09eR41